apps/src/templates/SortByNameDropdown.jsx

```diff
@@ -0,0 +1,32 @@
+import {useDispatch} from 'react-redux';
+import PropTypes from 'prop-types';
+import React from 'react';
+import i18n from '@cdo/locale';
+import {setSortByFamilyName} from '@cdo/apps/templates/currentUserRedux';
+
+function SortByNameDropdown({sortByStyles, selectStyles}) {
+  const dispatch = useDispatch();
+  return (
+    <div>
+      <div style={sortByStyles}>{i18n.sortBy()}</div>
+      <select
+        name="familyNameSort"
+        style={selectStyles}
+        defaultValue={i18n.displayName}
+        onChange={e => {
+          dispatch(setSortByFamilyName(e.target.value));
+        }}
+      >
+        <option value={false}>{i18n.displayName()}</option>
+        <option value={true}>{i18n.familyName()}</option>
```

Comment:
I don't know what we would add but is it possible we'd add any other options here? Just checking that a boolean makes the most sense to store for sorting, rather than an enum or something

Reply:
I was thinking about that, but because it's specific to *name sorting, I couldn't imagine what else we'd sort on. Happy to switch though just in case. Thoughts?

Reply:
Yeah that was my thinking too -- what else would we sort on? There's a mention in the spec about different ways to display names(ie `First Last`, `Last, First`, etc) but that feels different?

Reply:
Agreed. I think I'll leave it for now and we can always pivot later if we need another option